lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -471,6 +471,11 @@ Task terminateTask(String taskId)
    * Transfers a {@linkplain Task} to another {@linkplain Workbasket} while always setting
    * {@linkplain Task#isTransferred() isTransferred} to true.
    *
+   * @param taskId the {@linkplain Task#getId() id} of the {@linkplain Task} which should be
+   *     transferred
+   * @param destinationWorkbasketId the {@linkplain Workbasket#getId() id} of the target {@linkplain
+   *     Workbasket}
+   * @return the transferred {@linkplain Task}
```

Comment:
Isn't the boolean parameter missing?

Reply:
This transfer() function does not have the setTransferFlag() boolean parameter, but it calls another transfer() function that has the boolean parameter. It does not make sense to include the boolean parameter here, since it does not directly have the boolean parameter.